students/ChelseaSmith/Lesson4/katafourteen.py

```diff
@@ -0,0 +1,85 @@
+import random  # used further down to select seed values
+
+# making database
+
+def depunc(str):  # purging punctuation from a string
+    punct = "?!().;:,"
+    for x in str:
+        if x in punct:
+            str = str.replace(x, "")
+    return str
+
+
+# load data and put words into a list
+def texttolist(filename):
+    file = open(filename, "r")
+    words = []
+    for line in file:
+        phrase = line.rstrip().replace("--", " ").split()  # splits line of file into a list of strings. carriage return removed
+        for x in phrase:
+            y = depunc(x)  # removes punctuation
+            words.append(y)  # adds string y to list words
+    return words
+
+
+# check if two words are (already) a key in dictionary
+def indict(dictin, words):
+    tester = False
+    for x in dictin:
+        if x == words:
+            tester = True
+    return tester
```

Comment:
this function checks if `words` which is a key, exists in `dictin` which is a dictionary

you don't need a separate method for this, the code is simply `keyname in wordlot`